app/models/time_data/required_time_data.rb

```diff
@@ -10,6 +10,10 @@ def problem?
       true
     end
 
+    def problem_description_key
```

Comment:
`RequiredTimeData` should only happen on instrument orders where the reservation doesn't exist. There are two scenarios I can think of: 1) a bug (this happened recently, we're cleaning up most of the data, but there will still be a few complete orders with no reservation) 2) when you're creating an order in your cart (part of a bundle, adding multiple reservations at once, etc), the order detail is created, but the reservation doesn't exist yet.

So in the first case, we would hit this because the order is complete, and we would want to consider it a problem.

In the second case, I don't think we ever hit the `problem?` method (we do hit  `order_completable?`), but it's still useful for staying parallel with the other classes.